frontend/src/components/Calendar/ClickableDateContainer.tsx

```diff
@@ -38,6 +38,7 @@ export const ClickableDateContainer = (props: { date: number; eventPresent?: boo
 
   const DateContainerClickHandler = (newClickedDate: number) => {
     dispatch(SetClickedDate(newClickedDate))
```

Comment:
whats click date for? also it should be camelCase! 
```suggestion
    dispatch(setClickedDate(newClickedDate))
```